providers/netsuite/internal/suiteql/read.go

```diff
@@ -0,0 +1,114 @@
+package suiteql
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/urlbuilder"
+	"github.com/amp-labs/connectors/providers/netsuite/internal/shared"
+)
+
+const (
+	// SuiteQL timestamp format for TO_TIMESTAMP function. This seems to be a magic
+	// date format that is accepted by SuiteQL. We should verify it this actually
+	// works for all instances.
+	suiteQLTimestampFormat = "2006-01-02 15:04:05.000000000"
+
+	defaultLimit = 1000
```

Comment:
`maxRecords` or `pageSize` would be a clearer name, I had to look at how it's used in the code to understand what it actually means